src/app/clients/getClientParcelsData.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect, test } from "@jest/globals";
+import {
+    ExpandedClientParcelDetails,
+    ParcelsDetail,
+    rawDataToClientParcelsDetails,
+} from "@/app/clients/getClientParcelsData";
+
+const logID = "a2adb0ba-873e-506b-abd1-8cd1782923c8";
+
+jest.mock("@/logger/logger", () => ({
+    logErrorReturnLogId: jest.fn(() => Promise.resolve(logID)),
+}));
+
+jest.mock("@/supabaseClient", () => {
+    return { default: jest.fn() };
+});
+
+describe("test for getClientparcelsData", () => {
+    test("Should format the data correctly", () => {
```

Comment:
Can we use the `it(...` rather than `test`, it's a minor readability thing, but tends to be preferred

Reply:
this is done